xplique/attributions/image_object_detector_explainer.py

```diff
@@ -0,0 +1,214 @@
+"""
+Module related to Object detector method
+"""
+from typing import Iterable, Tuple, Union, Optional
+import abc
+import tensorflow as tf
+import numpy as np
+from xplique.attributions.base import BlackBoxExplainer
+from xplique.commons.tf_operations import inference_batching
+
+
+class IIouCalculator:
+    """
+    Used to compute the IOU.
+    """
+    @abc.abstractmethod
+    def intersect(self, obj1: tf.Tensor, obj2: tf.Tensor) -> tf.Tensor:
+        """
+        Compute the intersection between two batched objects
+        An object could be box, segmentation mask...
+        :param obj1: object1

```

Comment:
The documentation being generated with mkdocs, can you please comply to the documentation convention in the other files ? 
```
 """
 Compute the intersection between two batched objects
 An object could be a box, segmentation mask...
 
Parameters
------------
obj1
       object number 1. Can be a box, a segmentation masks, etc..
obj2
       object number 2.  Can be a box, a segmentation masks, etc..

Returns
--------
score
        A score between [0,1] corresponding to the intersection of the 2 objects.
"""
```

Reply:
Done.